calicoctl/commands/node/status.go

```diff
@@ -129,7 +136,19 @@ func drawTable(birdOut string) {
 
 		if s != "" {
 			col := []string{}
-			fields := strings.Fields(line)[3:6]
+			f := strings.Fields(line)[3:]
```

Comment:
This is all quite confusing without comments to describe what is happening.

What is `f` supposed to represent? Do you really need to copy this data around?